client/client/app/components/ngbBlastSearchPanel/ngbBlastWholeGenomeView/ngbBlastWholeGenomeView.scss

```diff
@@ -0,0 +1,17 @@
+md-dialog {
+  min-width: 800px;
+  min-height: 800px;
+  height: 800px;
```

Comment:
This will be appropriate not for all screen resolutions (please, check the **NOW** screenshot on the another comment).
Let's decrease minimum height to the 400px, but introduce height in *viewport-height* measurement (e.g., `80vh`)